src/NorthwindWebsite/Middleware/Handlers/Implementations/ImageCachingHandler.cs

```diff
@@ -0,0 +1,70 @@
+﻿using NorthwindWebsite.Core.ApplicationSettings;
+using NorthwindWebsite.Core.Constants;
+using NorthwindWebsite.Core.CustomExceptions.BusinessExceptions;
+using NorthwindWebsite.Middleware.Handlers.Interfaces;
+
+namespace NorthwindWebsite.Middleware.Handlers.Implementations;
+
+public class ImageCachingHandler : IImageCachingHandler
+{
+    private readonly AppSettings _appSettings;
+
+    private readonly string _cachingFolder;
+
+    public ImageCachingHandler(AppSettings appSettings)
+    {
+        _appSettings = appSettings;
+        _cachingFolder = string.Concat(
+            AppDomain.CurrentDomain.BaseDirectory,
+            _appSettings.CachingConfigs.CachingFolder);
+    }
+
+    public bool IsContained(string imageIndex)
+    {
+        var imageName = string.Concat(_cachingFolder, imageIndex, FileNameConstants.BmpExtension);
```

Comment:
I think that it can be moved to separate private static method, you can just pass one single argument - imageIndex